crates/uv-resolver/tests/resolver.rs

```diff
@@ -120,7 +119,10 @@ async fn resolve(
     let client = RegistryClientBuilder::new(Cache::temp()?).build();
     let flat_index = FlatIndex::default();
     let index = InMemoryIndex::default();
-    let interpreter = Interpreter::artificial(Platform::current()?, markers.clone());
+    // TODO(konstin): Should we also use the bootstrapped pythons here?
+    let real_interpreter =
```

Comment:
What's going on here?

Reply:
The fake interpreter needs to know it's platform to compute the tags we need for resolution. Previously, we could ask in rust, now we need an actual python interpreter. Assuming the dev has any python 3 with the right os + arch installed seemed like an ok assumption here, but i can pull in our bootstrapped python finding for a correct works-independent-of-outside-project-environment solution.